lib/knapsack_pro/runners/queue/rspec_runner.rb

```diff
@@ -165,6 +186,21 @@ def pull_tests_from_queue(can_initialize_queue: false)
           test_file_paths
         end
 
+        def print_current_rspec_batch_command
```

Comment:
suggestion

I'd keep current & batch and rspec & command together:
```suggestion
        def print_current_batch_rspec_command
```